src/Core/Mesh/TriangleMesh.hpp

```diff
@@ -106,24 +114,34 @@ class TriangleMesh {
         return m_vertexAttribs.addAttrib<T>( name );
     }
 
-    /// Remove attribute by handle, invalidate the handles to this attribute.
+    /// Remove attribute by handle.
     /// \see AttribManager::removeAttrib() for more info.
     template <typename T>
     void removeAttrib( AttribHandle<T> h ) {
         m_vertexAttribs.removeAttrib( h );
     }
 
-    /// Copy only the required attributes (deep copy).
-    /// The position and normal attributes are always copied, no need to provide their handles.
+    /// Erases all attributes, leaving the mesh with the topology and geometry only.
+    inline void clearAttributes();
+
+    /// Copy only the mesh topology and geometry.
+    /// The needed attributes can be copied through copyAttributes().
+    inline void copyBaseGeometry( const TriangleMesh& other );
+
+    /// Copy only the required attributes from \p input, keeping the old ones.
```

Comment:
I don't understand what means *keeping the old ones*.

Reply:
This comment must also describe what happens if the input handles have the same name than already existing attributes: is the attribute replaced ?

Reply:
1- By *keeping the old ones* I meant that the attributes present in the mesh bafore calling this method are still there after, but it might be misleading in some way, I will remove that
2- For existing attributes, they will be replaced, as mentioned in the doc for AttribManager which is referred to, but I can put it there and remove the link to AttribManager since it won't provide more info

Reply:
Ok, you can replace `keeping the old ones` by `existing attributes are kept untouched, except if overwritten by attributes copied from \p other.`